{{ cookiecutter.service_name }}/config/config_test.go

```diff
@@ -0,0 +1,8 @@
+package config
+
+import (
+	"testing"
+)
+
+func TestNew(t *testing.T) {
+}
```

Comment:
Add comment inside this function that says
// Use package gotest to generate this function based on your config structs